core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -367,13 +367,7 @@ final case class App(id: AppId,
   def getProxyUrls(project: GoogleProject, proxyUrlBase: String): Map[ServiceName, URL] =
     appResources.services.map { service =>
       val proxyPath = s"google/v1/apps/${project.value}/${appName.value}/${service.config.name.value}"
-      appType match {
-        case Galaxy | Custom =>
-          (service.config.name, new URL(s"${proxyUrlBase}${proxyPath}"))
-        case Cromwell =>
-          (service.config.name,
-           new URL(s"${proxyUrlBase}${proxyPath}/swagger/index.html?url=/proxy/${proxyPath}/swagger/cromwell.yaml"))
-      }
+      (service.config.name, new URL(s"${proxyUrlBase}${proxyPath}"))
```

Comment:
Does this work without the trailing `/`? I think I remember seeing one last 301 to add the `/`, but maybe that one works.

Reply:
Testing locally, the paths seem to work equivalently in my browser. However, for the benefit of my `workspace-cromwell` script which probably assumes a trailing `/`, I I will re-add it